diff --git a/CMakeLists.txt b/CMakeLists.txt
index 68b65bb..98a1ab0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,7 +65,8 @@ ENDIF(WIN32)
 # ----------------------------------------------------
 # --- DEPENDENCIES -----------------------------------
 # ----------------------------------------------------
-ADD_REQUIRED_DEPENDENCY("eigen3 >= 3.0.5")
+# ADD_REQUIRED_DEPENDENCY("eigen3 >= 3.0.5")
+SET(EIGEN3_INCLUDE_DIRS "${CMAKE_INSTALL_PREFIX}/include/eigen3")
 
 SET(BOOST_COMPONENTS python)
 SEARCH_FOR_BOOST()
